src/Arcus.Templates.Tests.Integration/AzureFunctions/Http/AzureFunctionsHttpProjectOptions.cs

```diff
@@ -23,28 +19,26 @@ public AzureFunctionsHttpProjectOptions()
         /// </summary>
         public AzureFunctionsHttpProjectOptions WithIncludeHealthChecks()
         {
-            ProjectOptions newOptions = AddOption("--include-healthchecks");
```

Comment:
Stepped away from immutability bc it is harder to do in C# (even with records, which are just light-weight F# records), and since the added-value here is none.